src/lightning/pytorch/utilities/data/dataset.py

```diff
@@ -0,0 +1,78 @@
+import os
+import tempfile
+from abc import ABC, abstractmethod
+from typing import Any, Optional, Tuple
+
+from torch.utils.data import Dataset as TorchDataset
+
+from lightning.pytorch.utilities.data.fileio import *
```

Comment:
ideally, we don't import `*` as this might also import imported modules and could (in theory) lead to unexpected behavior. Usually we try to be explicit with our imports :)

Reply:
fixed